### PR TITLE
ZCS-12956 WCAG Re-opened Issues Part 4

### PR DESCRIPTION
**Pull request by miteshsavani810**
- Added hidden table such that screen reader can navigate it through shortcut and user can navigate it's column and row cell.

### CODE REVIEW

**Review by HimanshuSojitraSynacor — APPROVED** [review]

**Review by HimanshuSojitraSynacor — APPROVED** [review]

File: WebRoot/js/ajax/dwt/widgets/DwtListView.js [hunk 1]
```diff
@@ -47,6 +47,13 @@ DwtListView = function(params) {
 	params.className = params.className || "DwtListView";
 	DwtComposite.call(this, params);
 
+	if (this.tableView && params.headerList) {
+		this.column = [];
+		for (var i = 0; i < params.headerList.length; i++) {
```
Comment (lavegupta) [inline WebRoot/js/ajax/dwt/widgets/DwtListView.js]: `for (var i = 0, length = params.headerList.length; i < length; i++) {`

**Review by silentsakky — APPROVED** [review]

File: WebRoot/js/ajax/dwt/widgets/DwtListView.js [hunk 2]
```diff
@@ -508,6 +515,55 @@ function(defaultColumnSort, noResultsOk) {
...
+
+	var div = this.getHtmlElement();
+	var table = div.querySelector('table.ScreenReaderOnly');
+	if (table) {
+		table.remove();
+	}
+	var newTable = document.createElement('table');
+	newTable.className = 'ScreenReaderOnly';
+	var header = '<thead class="ScreenReaderOnly"><tr>';
+	for (var i = 0; i < this.column.length; i++) {
```
Comment (lavegupta) [inline WebRoot/js/ajax/dwt/widgets/DwtListView.js]: `for (var i = 0, length = this.column.length; i < length; i++) {`

File: WebRoot/js/ajax/dwt/widgets/DwtListView.js [hunk 3]
```diff
@@ -508,6 +515,55 @@ function(defaultColumnSort, noResultsOk) {
...
+		header += '<th>' + this.column[i] + '</th>';
+	}
+	header += '</tr></thead>';
+
+	var body = '<tbody class="ScreenReaderOnly">';
+
+	// Get list of rows from table appear on UI.
+	var uiRows = div.querySelector('div.DwtListView-Rows').childNodes;
+
+	for (var j = 0; j < uiRows.length; j++) {
```
Comment (lavegupta) [inline WebRoot/js/ajax/dwt/widgets/DwtListView.js]: same as above

File: WebRoot/js/ajax/dwt/widgets/DwtListView.js [hunk 4]
```diff
@@ -508,6 +515,55 @@ function(defaultColumnSort, noResultsOk) {
...
+		
+		var columnTds = uiRows[j].querySelectorAll('td');
+		if (columnTds && columnTds.length === this.column.length) {
+			body +='<tr>'; 
+			for (var k = 0; k < columnTds.length; k++) {
```
Comment (lavegupta) [inline WebRoot/js/ajax/dwt/widgets/DwtListView.js]: same as above

File: WebRoot/js/ajax/dwt/widgets/DwtListView.js (hunk 1 above)
Comment (miteshsavani810) [reply WebRoot/js/ajax/dwt/widgets/DwtListView.js]: Done.

File: WebRoot/js/ajax/dwt/widgets/DwtListView.js (hunk 2 above)
Comment (miteshsavani810) [reply WebRoot/js/ajax/dwt/widgets/DwtListView.js]: Done.

File: WebRoot/js/ajax/dwt/widgets/DwtListView.js (hunk 3 above)
Comment (miteshsavani810) [reply WebRoot/js/ajax/dwt/widgets/DwtListView.js]: Done.

File: WebRoot/js/ajax/dwt/widgets/DwtListView.js (hunk 4 above)
Comment (miteshsavani810) [reply WebRoot/js/ajax/dwt/widgets/DwtListView.js]: Done.

**Review by lavegupta — APPROVED** [review]

**Pull request merged**